### PR TITLE
A first pass at creating a molecule test which requires JWS

### PR DESCRIPTION
**Pull request by csutherl**
I had to add a few small fixes for the tomcat-vault feature (which I haven't tested yet) which are included here as well. I'm not sure if I'm on the right track, so I thought I'd push what I have and get a review. Thanks!

### CODE REVIEW

**Comment by csutherl** [comment]
@rpelisse OK, this latest commit (22f9129) works using some sample files (keystore, properties, data file) so it's ready for review. This iteration requires users to provide a vault that's already configured along with the properties file and data file (which we should probably make optional since it's not required). I was also thinking about adding a new switch like `tomcat.vault.init` which will execute the call to `keytool` and the vault init that was there before. After the init, users would have to encrypt their data and copy the appropriate strings into the config files. Thoughts?

File: roles/jws/tasks/main.yml [hunk 1]
```diff
@@ -136,16 +137,29 @@
...
     - block:
-        - name: "Copy {{ tomcat.service.hr_name }} vault file from control node to remote"
+        - name: "Copy {{ tomcat.service.hr_name }} vault keystore from control node to remote"
           copy:
             src: "{{ tomcat.vault.name }}"
-            dest: "{{ tomcat.home }}/"
+            dest: "{{ tomcat.home }}"
+            owner: "{{ tomcat.user }}"
+            group: "{{ tomcat.group }}"
+            mode: 0750
```
Comment (guidograzioli) [inline roles/jws/tasks/main.yml:146]: this (and below) should be 0640 or maybe 0440

File: roles/jws/templates/catalina.properties.j2 [hunk 2]
```diff
@@ -0,0 +1,214 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (guidograzioli) [inline roles/jws/templates/catalina.properties.j2:1]: Please add on first line:

    # {{ ansible_managed }}

File: roles/jws_validation/tasks/main.yml [hunk 3]
```diff
@@ -45,3 +45,8 @@
     - tomcat_listen_ajp_enabled is defined
     - tomcat_listen_ajp_bind_address is defined
     - tomcat_listen_ajp_port is defined
+
+- name: "Check tomcat-vault configuration"
+  include_tasks: check_tomcat_vault.yml
+  when:
+    - tomcat_vault_enable == True
```
Comment (guidograzioli) [inline roles/jws_validation/tasks/main.yml:52]: no need for ``== True`` here, just ```when: tomcat_vault_enabled```

**Review by guidograzioli — CHANGES_REQUESTED** [review]

**Comment by csutherl** [comment]
This test has to live elsewhere :/

**Pull request closed without merging**